server/middleware/setUpHealthChecks.ts

```diff
@@ -1,39 +1,22 @@
 import express, { Router } from 'express'
 
-import healthcheck from '../services/healthCheck'
+import { monitoringMiddleware, endpointComponent, EndpointComponentOptions } from '@ministryofjustice/hmpps-monitoring'
 import type { ApplicationInfo } from '../applicationInfo'
+import logger from '../../logger'
 
-export default function setUpHealthChecks(applicationInfo: ApplicationInfo): Router {
-  const router = express.Router()
+type ApiToMonitor = [name: string, config: EndpointComponentOptions]
 
-  router.get('/health', (req, res, next) => {
-    healthcheck(applicationInfo, result => {
-      if (result.status !== 'UP') {
-        res.status(503)
-      }
-      res.json(result)
-    })
-  })
+export default function setUpHealthChecks(applicationInfo: ApplicationInfo, components: ApiToMonitor[]): Router {
```

Comment:
Where does type `ApiToMonitor` come from? Is it missing from this PR? 😃

Reply:
it's on line 7^

Reply:
Tidied this up to remove the extra type